src/PostgREST/Types.hs

```diff
@@ -398,8 +407,26 @@ type Alias = Text
 type Cast = Text
 type NodeName = Text
 
--- Rpc query param, only used for GET rpcs
-type RpcQParam = (Text, Text)
+-- RPC query param, used for POST of form-data and GET requests
+data RpcParamValue = Fixed Text | Variadic [Text]
+
+mergeParams :: RpcParamValue -> RpcParamValue -> RpcParamValue
+mergeParams (Variadic a) (Variadic b) = Variadic $ b ++ a
+-- repeated params for non-variadic arguments are not merged
+mergeParams v _                       = v
```

Comment:
need to add a test here, that it keeps the first occurence of a repeated param in the non-variadic case - not the last.